lib/account/pages/reviews_page.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+import 'package:motis_mitfahr_app/util/big_button.dart';
+import 'package:motis_mitfahr_app/util/profiles/profile_widget.dart';
+import 'package:motis_mitfahr_app/util/profiles/reviews/aggregate_review_widget.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../util/review_detail.dart';
+import '../models/review.dart';
+import 'write_review_page.dart';
+
+class ReviewsPage extends StatefulWidget {
+  final int profileId;
+  final Profile? profile;
+
+  const ReviewsPage({super.key, required this.profileId}) : profile = null;
```

Comment:
Do we really need this? because there should not be any calls to the reviewPage without a Profile or am I missing something?

Reply:
Its necessary for the anonymous search. Since they can look at the Ratings by going from SearchCard to the Ride Detail and then to the Rating.

Reply:
shouldn't there the Profile of the Person who was Reviewed been loaded?

Reply:
@istzen is probably right, but this is just to be consistent with the other pages and to maybe support that in the future